docs/configuration/config-file-reference.md

```diff
@@ -4371,6 +4375,18 @@ ring:
 # Disable the rule_group label on exported metrics
 # CLI flag: -ruler.disable-rule-group-label
 [disable_rule_group_label: <boolean> | default = false]
+
+# Enable high availability
+# CLI flag: -ruler.enable-ha
+[enable_ha: <boolean> | default = false]
+
+# Timeout for fanout calls to other rulers
+# CLI flag: -ruler.list-rules-fanout-timeout
+[list_rules_fanout_timeout: <duration> | default = 2m]
+
+# Timeout for liveness checks performed during rule sync
+# CLI flag: -ruler.liveness-check-timeout
+[liveness_check_timeout: <duration> | default = 1s]
```

Comment:
Is 1s a good default here?

Reply:
This is a simple/fast check. I thought `1s` might be big enough for default. If RF=2, this should check only one Ruler. If RF=3, it will check 2 Rulers. Do you think it needs to be higher/lower?

Reply:
Do we need this config at all? We cannot just put a sensible timeout?